scripts/main.py

```diff
@@ -355,7 +366,7 @@ def parse_arguments():
             f"COMMAND = {Fore.CYAN}{command_name}{Style.RESET_ALL}  ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}")
 
     # Execute command
-    if command_name.lower() == "error":
```

Comment:
This is an unrelated change

Reply:
done, had to pull latest into my branch.